doc/rbd/rbd-live-migration.rst

```diff
@@ -176,6 +177,19 @@ it utilizes native Ceph operations. For example, to import from the image
             "snap_name": "snap1"
         }
 
+The ``qcow`` format can be used to describe a QCOW (QEMU copy-on-write) block
+device. Only the original QCOW (v1) format is currently supported, but support
```

Comment:
Is anyone still using qcow1? I'm not sure there's any benefit to supporting it these days. It'll just add dead code to the project? I'm surprised qemu-img still supports it properly (I know you can't do consistency check on it, for example).

Reply:
Feel free to talk to me offline about software but this is not "[adding] dead code" to Ceph. QCOW and QCOW2 share essentially 100% code on the read IO path.